README.md

```diff
@@ -91,6 +91,7 @@ Building metafacture-core from source is easy. All you need is git and JDK 11:
     ```bash
     $ ./gradlew publishToMavenLocal
     ```
+   In case you are building locally from a tag and you have no signing credentials you need to `touch .temp` before `./gradlew publishToMavenLocal` otherwise it will try to build a release and break due to missing signing.
```

Comment:
```suggestion
   In case you are building locally from a tag and you have no signing credentials you can use a workaround: just make sure to have an unclean status e.g. by doing `touch .temp` before doing `./gradlew publishToMavenLocal`. Otherwise it will try to build a release and break due to missing signing.
```